src/ert/scheduler/__init__.py

```diff
@@ -38,6 +38,8 @@ def create_driver(config: QueueConfig) -> Driver:
             bsub_cmd=queue_config.get("BSUB_CMD"),
             bkill_cmd=queue_config.get("BJOBS_CMD"),
             bjobs_cmd=queue_config.get("BJOBS_CMD"),
+            queue_name=queue_config.get("LSF_QUEUE"),
```

Comment:
Is it a bug that we did not pass in LSF_QUEUE already? (if so, it should probably be a separate PR, with some testing)

Reply:
It seems like it. I will split this commit out into a new PR, and create an issue for it.